SPECS/kernel-azure/config

```diff
@@ -17,6 +17,7 @@ CONFIG_CC_HAS_ASM_GOTO_OUTPUT=y
 CONFIG_CC_HAS_ASM_GOTO_TIED_OUTPUT=y
 CONFIG_CC_HAS_ASM_INLINE=y
 CONFIG_CC_HAS_NO_PROFILE_FN_ATTR=y
+CONFIG_PAHOLE_VERSION=121
```

Comment:
Result of this change https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/3597fd5f9217d6b7c24f3aefea629484a5505764